src/ImageService.Data/ImageProjectRepository.cs

```diff
@@ -0,0 +1,55 @@
+﻿using LT.DigitalOffice.ImageService.Data.Interfaces;
+using LT.DigitalOffice.ImageService.Data.Provider;
+using LT.DigitalOffice.ImageService.Models.Db;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+
+namespace LT.DigitalOffice.ImageService.Data
+{
+    public class ImageProjectRepository : IImageProjectRepository
+    {
+        private readonly IDataProvider _provider;
+
+        public ImageProjectRepository(IDataProvider provider)
+        {
+            _provider = provider;
+        }
+
+        public List<Guid> Create(List<DbImagesProject> imagesProjects)
+        {
+            if (imagesProjects.Contains(null))
+            {
+                throw new ArgumentNullException(nameof(imagesProjects));
+            }
+
+            _provider.ImagesProjects.AddRange(imagesProjects);
+            _provider.Save();
+
+            return imagesProjects.Select(x => x.Id).ToList();
+        }
+
+        public bool Delete(List<DbImagesProject> imagesProjects)
```

Comment:
why do you not accept List of ids? you can get dbImagesProject right here, by ids.

Reply:
done